Testscripts/Linux/perf_ntttcp.sh

```diff
@@ -188,6 +181,28 @@ Get_Cyclesperbytes()
 	echo "$cyclesperbytes"
 }
 
+Get_concreatedtime()
+{
+	concreatedtime_us=0
+	concreatedtime_us=$(grep "connections created in" "${1}" | tr ":" " " | awk '{ print $(NF-1) }')
+	if [[ ! $concreatedtime_us ]];
+	then
+		concreatedtime_us=0
+	fi
+	echo  "$concreatedtime_us"
+}
+
+Get_retranssegs()
+{
+	retrans_segs=0
```

Comment:
The same here. It seems to be taken care of it below.

Reply:
fixed